chore: tidy up makefile (DEV-1166) (#223)
dasch-swiss/dsp-tools: README.md

## Contributing to the documentation
The documentation is a collection of [markdown](https://en.wikipedia.org/wiki/Markdown) files in the `docs` folder.  
After updates of the files, build and check the result with the following command:

```bash
make docs-serve 
```

> The documentation is served at https://docs.dasch.swiss/latest/DSP-TOOLS. During the centralized release process of all

Reviewer: ```suggestion
The documentation is published on https://docs.dasch.swiss/latest/DSP-TOOLS. During the centralized release process of all
```

> programs of the DSP software stack, the docs of dsp-tools get built from the main branch to https://docs.dasch.swiss.

Reviewer: ```suggestion
components of the DSP software stack, the docs of dsp-tools get built from the main branch to https://docs.dasch.swiss.
```
